bin/libs/certificate.sh

```diff
@@ -883,9 +883,11 @@ EOF
     racf_connect2="s/^ \+LABEL[(]'certlabel'[)].*\$/            LABEL('${connect_label}') +/"
   fi
 
-  # used by ACF2
-  # TODO: how to pass this?
-  stc_group=
+  # used by ACF2  
+  stc_group=$(read_yaml "${ZWE_CLI_PARAMETER_CONFIG}" ".zowe.setup.security.groups.stc")
+  if [ -z "${security_groups_stc}" ]; then
```

Comment:
Is this correct? The variable 1 line above is `stc_group` and then `security_groups_stc` is being checked.